src/main/resources/assets/pyrotech/patchouli_books/book/zh_cn/entries/survive/campfire_effects.json

```diff
@@ -1,45 +1,45 @@
 {
-  "name": "Campfire Effects",
+  "name": "篝火效果",
   "icon": "pyrotech:campfire",
   "category": "survive",
   "pages": [
     {
       "type": "image",
-      "title": "Campfire Effects",
+      "title": "篝火效果",
       "border": false,
       "images": [
         "pyrotech:textures/gui/book/photo/campfire_effects_progression_00.png"
       ]
     },
     {
       "type": "campfire_effect",
-      "title": "Comfortable",
+      "title": "舒适",
       "image": "pyrotech:textures/potions/comfort.png",
-      "text": "Stand within range of a campfire at night to gain the $(thing)Comfortable$() effect.$(p)$(info) The $(thing)Comfortable$() effect increases the effect of eating.$(p)$(warning) This effect will disappear if hostile mobs are nearby."
+      "text": "在晚上时，站在篝火旁一定范围来获得$(thing)舒适$()效果。$(p)$(info)$(thing)舒适$()效果使进食的回复效果更好。$(p)$(warning)如果附近有怪物在游荡，这个效果就会消失。"
     },
     {
       "type": "campfire_effect",
-      "title": "Well Fed",
+      "title": "吃饱喝足",
       "image": "pyrotech:textures/potions/well_fed.png",
-      "text": "Eat a lot of food with the $(thing)Comfortable$() effect to gain the $(thing)Well Fed$() effect.$(p)$(info) The $(thing)Well Fed$() effect increases your stamina.$(p)$(warning) This effect will disappear over time."
```

Comment:
what does the "stamina" means?

Reply:
In this context stamina refers to the rate at which a player becomes hungry while performing tasks that reduce their hunger haunches. When the player has the `Well Fed` effect, the rate they become hungry is reduced and it takes longer for them to get hungry while performing tasks.